exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -66,21 +67,12 @@ func (b *Bitfinex) WsSend(data interface{}) error {
 	if err != nil {
 		return err
 	}
-	return b.WebsocketConn.WriteMessage(websocket.TextMessage, json)
-}
-
-// WsSubscribe subscribes to the websocket channel
-func (b *Bitfinex) WsSubscribe(channel string, params map[string]string) error {
-	req := make(map[string]string)
-	req["event"] = "subscribe"
-	req["channel"] = channel
-
-	if len(params) > 0 {
-		for k, v := range params {
-			req[k] = v
-		}
+	if b.Verbose {
+		log.Debugf("Sending message to websocket %v", data)
 	}
-	return b.WsSend(req)
+	// Basic rate limiter
+	time.Sleep(bitfinexWebsocketRateLimit)
```

Comment:
A rate limiter will need to be made for websocket, if two or more websocket events are sent at the time time, then this time.Sleep will effectively do nothing as the requests will be sent at the same time after the sleeper